lib.php

```diff
@@ -52,3 +48,11 @@ function tool_excimer_after_config(): void {
 function tool_excimer_performance_checks(): array {
     return [new slowest()];
 }
+
+/**
+ * Hook to run plugin before session start.
+ */
+function tool_excimer_before_session_start() {
+    // Start plugin.
+    tool_excimer_after_config();
```

Comment:
because you have refactored it I would call this the same here manager::create_if_needed();

Reply:
I'd also move them side by side, or better yet in the order they will be executed so this one first

Reply:
Updated